packages/extension-mention/src/mention.ts

```diff
@@ -17,9 +20,16 @@ export const Mention = Node.create<MentionOptions>({
   addOptions() {
     return {
       HTMLAttributes: {},
-      renderLabel({ options, node }) {
+      renderText({ options, node }) {
```

Comment:
![image](https://github.com/ueberdosis/tiptap/assets/896179/a25a720f-a3d4-4a50-a355-38d38f84ffbe)

not sure what exactly is the issue here, but when not passing `render*` methods, the mentions extension renders this, somehow caused by a duplicated span element.

Reply:
fixed